packages/manager/apps/pci-private-registry/src/pages/list/IAMColumnLabel.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+  OsdsIcon,
+  OsdsPopover,
+  OsdsPopoverContent,
+  OsdsText,
+} from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import {
+  ODS_TEXT_SIZE,
+  ODS_TEXT_LEVEL,
+  ODS_ICON_NAME,
+  ODS_ICON_SIZE,
+} from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+
+const IAMColumnLabel = () => {
+  const { t } = useTranslation();
+
+  return (
+    <span className="inline-flex">
+      <OsdsText
+        data-testid="registryIamColumnLabel"
+        className="text-[--ods-color-primary-800] self-center"
+        size={ODS_TEXT_SIZE._500}
+      >
+        {t('private_registry_iam_authentication_status')}
+      </OsdsText>
+      <span onClick={(event) => event.stopPropagation()}>
```

Comment:
We could improve the HTML semantic as `span` should not used as a container purpose :)